src/people/widgetViews/summaries/wantedSummaries/CodingBounty.tsx

```diff
@@ -450,6 +450,8 @@ function MobileView(props: CodingBountiesProps) {
     );
   }
 
+  const isDisabled = !(isAssigned || paid) ? false : true;
```

Comment:
not sure why the turnary we can just do this right?
`const isDisabled = isAssigned || paid`

Reply:
It's true... Newbie code 😅